javascript/javascript3/week3/Shopping cart/style.css

```diff
@@ -0,0 +1,122 @@
+* {
+  box-sizing: border-box;
+}
+
+body,
+main {
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-family: "Montserrat", sans-serif;
+  padding-top: 4%;
+  background-color: rgb(202, 226, 202);
+}
+
+header {
+  width: 100%;
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.currency {
+  float: right;
+}
+
+.logo {
+  width: 120px;
+}
+
+.logo img {
+  width: 120%;
+}
+
+.title {
+  font-family: "Rouge Script", cursive;
+  font-size: 40px;
+  margin: 0 0 50px;
+  text-align: center;
+  text-shadow: 2px 3px 3px #082b34;
+}
+
+label {
+  margin-top: 16px;
+  margin-bottom: 16px;
+}
+
+#product-options {
+  width: 300px;
+  padding: 7px;
+  border: 1px solid #ccc;
+  border-radius: 6px;
+  text-align: center;
+  font-weight: bold;
+  box-shadow: 3px 3px 3px #787e80;
+}
+
+#currency {
+  width: 80px;
+  padding: 7px;
+  border: 1px solid #ccc;
+  border-radius: 6px;
+  text-align: center;
+  font-weight: bold;
+  box-shadow: 3px 3px 3px #787e80;
+}
+
+button {
+  background-color: #142a35;
+  color: white;
+  padding: 12px 20px;
+  border: none;
+  border-radius: 4px;
+  cursor: pointer;
+  margin-top: 5%;
+  margin-bottom: 5%;
+  cursor: pointer;
+  box-shadow: 3px 3px 3px #787e80;
+}
+
+ul {
+  list-style-type: none;
+  border: 1px solid var(--plum);
+  padding: 0;
+  width: 500px;
+}
+
+#username {
+  font-family: "Rouge Script", cursive;
+  font-size: larger;
+  margin: 0 0 50px;
+  text-align: center;
+  color: green;
+}
+
+.list-item {
+  white-space: pre-line;
+  margin-top: 16px;
+  margin-bottom: 16px;
+  margin-left: 16px;
+  margin-right: 16px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+section.Shopping-basket {
```

Comment:
you do not need to mention section here because with this class name also u can reflect your style.